standup/script.js

```diff
@@ -121,8 +121,34 @@ function createTableHeaderElement() {
     type: 'th',
     classList: ['user', 'date', 'table-head'],
   });
-  headerCellElement.innerHTML = 'DATES ➡️<hr />USERS ⬇️';
+
+  const dateTextNode = document.createTextNode('DATES➡️');
+  const breakTag = document.createElement('br');
+  const breakTag2 = document.createElement('br');
```

Comment:
Why for the same tag are you creating two variable?

Reply:
@vinit717 
I have tried using same element twice but it was not applying two br tags there .So had to make 2 for same ui.

Reply:
@vinit717 i have researched regrading this and the answer I found is this.
The reason we're seeing only one <br> tag reflected in a container, even though you are appending two, might be related to how HTML handles consecutive line breaks. Browsers tend to treat multiple <br> tags consecutively as a single break in some cases due to rendering optimizations.

Do you want me to remove it or leave it as it is now?

Reply:
Done with the changes